src/main/java/com/mcmoddev/lib/integration/plugins/TinkersConstructBase.java

```diff
@@ -31,10 +28,13 @@
 
 	@Override
 	public void init() {
-		if (initDone || !Options.isModEnabled(PLUGIN_MODID)) {
+		if (initDone/* || !Options.isModEnabled(PLUGIN_MODID)*/) {
 			return;
 		}
 
+		this.modifierSetup();
+		this.modifierRegister();
+

```

Comment:
Yeah... however... there should have been a plugin done basing off this (this is the base class and really shouldn't do much of anything on its own) - beyond that, see recent BMe, as Tinkers' API and timing requirements are strange and not fun to figure out.

Reply:
this is just temp code... I will probably just leave the new modifier in and remove everything else... then we can do it properly when it gets integrated with new versions from BMs and such.